pkgs/development/python-modules/tornado/default.nix

```diff
@@ -6,18 +6,18 @@
 , backports_ssl_match_hostname
 , certifi
 , singledispatch
-, pythonOlder
 , futures
-, version ? "5.1"
+, isPy27
+, version ? "6.0.4"
 }:
 
 let
   versionMap = {
     "4.5.3" = {
       sha256 = "02jzd23l4r6fswmwxaica9ldlyc2p6q8dk6dyff7j58fmdzf853d";
     };
-    "5.1" = {
```

Comment:
we may want to keep around a v5 for applications which aren't ready to upgrade to 6